### PR TITLE
fix(billing): Fix plural of profileDuration, remove default

### PR DESCRIPTION
**Pull request by scttcper**
Change constant, add comment explaining when plural vs DataCategory would be used

### CODE REVIEW

File: static/app/constants/index.tsx [hunk 1]
```diff
@@ -329,7 +329,7 @@ export const DATA_CATEGORY_INFO = {
   [DataCategoryExact.PROFILE_DURATION]: {
     name: DataCategoryExact.PROFILE_DURATION,
     apiName: 'profileSeconds',
-    plural: 'profileSeconds',
+    plural: 'profileDuration',
     displayName: 'profile seconds',
     titleName: t('Profile Seconds'),
```
Comment (vbro) [inline static/app/constants/index.tsx:334]: The apiName is `profileDuration` and we display `profile hours`.
See https://github.com/getsentry/getsentry/blob/master/getsentry/billing/utils/data_category.py#L209-L218

File: static/app/types/core.tsx [hunk 2]
```diff
@@ -67,12 +67,11 @@ export type Choice = [
...
 /**
- * This legacy type used plurals which will cause compatibility issues when categories
- * become more complex, e.g. processed transactions, session replays. Instead, access these values
- * with `DATA_CATEGORY_INFO[category].plural`, where category is the `DataCategoryExact` enum value.
+ * These are very similar to the plural types of DATA_CATEGORY_INFO.
+ * DATA_CATEGORY_INFO and DataCategoryExact have additional categories
+ * that are used in stats but not other places like billing.
  */
 export enum DataCategory {
-  DEFAULT = 'default',
```
Comment (vbro) [inline static/app/types/core.tsx:75]: What is the default for?

Comment (scttcper) [reply static/app/types/core.tsx:75]: it seems unused, but we can keep it for now

Comment (markstory) [reply static/app/types/core.tsx:75]: We have a DataCategory called DEFAULT. It gets used for error events that don't have stack traces like from `capture_message`. Default events consume error quota.

Comment (vbro) [reply static/app/types/core.tsx:75]: Perhaps we should investigate further and remove `DataCategory.DEFAULT` in a separate PR if necessary.

**Review by vbro — APPROVED** [review]

**Pull request merged**